### PR TITLE
[Fix] argocd install

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr fixes the sync of liqo with argocd by removing the pre/post install/update helm hooks not correctly handled by argocd

### CODE REVIEW

File: pkg/consts/controllers.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ const (
 	CtrlPodInternalNet         = "pod_internalnet"
 	CtrlPublicKey              = "publickey"
 	CtrlRouteConfiguration     = "routeconfiguration"
+	CtrlSecretWebhook          = "secret_webhook"
```
Comment (fra98) [inline pkg/consts/controllers.go:54]: Can you move this to the Core section?

Comment (aleoli) [reply pkg/consts/controllers.go:54]: Yes, sorry, i focused on the alphabetical order and missed the group 😅

**Pull request merged**